src/v3/test/integration/identify-with-password-error-flow.test.tsx

```diff
@@ -71,6 +71,7 @@ describe('identify-with-password-error-flow', () => {
           status: 200,
         },
       },
+      widgetOptions: { features: { rememberMe: false } },
```

Comment:
Just curious, why was this needed? 🤔

Reply:
If you see this code in `transformIdentify` https://github.com/okta/okta-signin-widget/blob/8beb7299d230864090837133fd30ffc7b600f7bd/src/v3/src/transformer/identify/transformIdentify.ts#L53-L58
we actually default `rememberMe` to true if it's undefined or falsey

Reply:
Ohh, that makes sense, I forgot that feature is defaulted to true in g2 and we wanted to preserve it 🙏